src/rpc/misc.cpp

```diff
@@ -199,37 +199,39 @@ UniValue deriveaddresses(const JSONRPCRequest& request)
             "For more information on output descriptors, see the documentation in the doc/descriptors.md file.\n"},
             {
                 {"descriptor", RPCArg::Type::STR, RPCArg::Optional::NO, "The descriptor."},
-                {"begin", RPCArg::Type::NUM, RPCArg::Optional::OMITTED_NAMED_ARG, "If a ranged descriptor is used, this specifies the beginning of the range to import."},
-                {"end", RPCArg::Type::NUM, RPCArg::Optional::OMITTED_NAMED_ARG, "If a ranged descriptor is used, this specifies the end of the range to import."}
+                {"range", RPCArg::Type::RANGE, RPCArg::Optional::OMITTED_NAMED_ARG, "If a ranged descriptor is used, this specifies the end or the range (in [begin,end] notation) to derive."},
```

Comment:
bitcoin-cli users incoming 😄

Reply:
How so?

Reply:
BTW should update convert params in client.cpp.

With the shell it must be `bitcoin-cli ... `'[0,100]'` right?

Reply:
I don't think so.

Reply:
Yes, I have to test that.

https://github.com/bitcoin/bitcoin/blob/a0d4e79b4dbbfda027054c991705313fcf04135c/src/rpc/client.cpp#L71

This must be updated.

Reply:
Right, done!